.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+# The secrets.VAR_NAME values are defined in the
```

Comment:
This is currently deploying to my personal account and mainly intended for temporary use while we were working on this large PR. I'm not sure how the current site gets deployed, but I'm sure this could get modified to push to the correct site.